src/ringcentral-web-phone.js

```diff
@@ -369,8 +394,8 @@
             session.removeListener('cancel', stopPlaying);
             session.removeListener('failed', stopPlaying);
             session.removeListener('replaced', stopPlaying);
-            session.mediaHandler.removeListener('iceConnectionCompleted', stopPlaying);
-            session.mediaHandler.removeListener('iceConnectionFailed', stopPlaying);
+            // session.sessionDescriptionHandler.removeListener('iceConnectionCompleted', stopPlaying);
+            // session.sessionDescriptionHandler.removeListener('iceConnectionFailed', stopPlaying);
```

Comment:
Remove if it's not needed, don't leave it commented

Reply:
Removed

Reply:
I have asked to remove! Not uncomment!